### ISSUE TITLE
`@stylistic/declaration-block-trailing-semicolon` passes over the last declaration of a block where a comment closes the block behind it

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#217]
Found by the review of [#208](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/208), which is the mirror image: the four `declaration-block-semicolon-*` rules read a comment closing the block as the block's last node and measured a declaration for a semicolon it does not hold, while this rule reads it the same way and passes over the declaration it ought to measure.

### What minimal example or steps are needed to reproduce the bug?

```css
a { color: pink /* c */ }
```

and, with `never`:

```css
a { color: pink; /* c */ }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/declaration-block-trailing-semicolon": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json
```

### What did you expect to happen?

`1:15 Expected a trailing semicolon` for the first file, as `a { color: pink }` is reported, and `Unexpected trailing semicolon` for the second under `never`, as `a { color: pink; }` is — the comment behind the declaration changes nothing about the semicolon.

### What actually happened?

Nothing is reported, under either option, and `--fix` leaves both files as they are. The same holds for a bodiless at-rule closing a block with a comment behind it, `a { @include x /* c */ }`.

### What is the cause?

The rule measures only the node that is `parent.last`, and a comment standing between the last declaration and the closing brace is a node of its own, the last one, so the declaration is never looked at. The block's `raws.semicolon`, which the rule reads, speaks of the last node that is not a comment; the declaration wants to be found the same way, past the comments. [#208](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/208) gives the four `declaration-block-semicolon-*` rules a helper for exactly that reading.
